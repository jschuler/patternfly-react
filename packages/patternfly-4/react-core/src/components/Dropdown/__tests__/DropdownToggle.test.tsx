// Auto-generated do not edit

import * as React from 'react';
import { shallow } from 'enzyme';
import { DropdownToggle } from '../DropdownToggle';
// any missing imports can usually be resolved by adding them here
import {  } from '..';

describe('DropdownToggle test', () => {
  it('DropdownToggle should match snapshot', () => {
    const view = shallow(
      <DropdownToggle
        id={"''"}
				children={<>ReactNode</>}
				className={"''"}
				isOpen={false}
				onToggle={(_isOpen: boolean) => undefined as any}
				parentRef={null}
				isFocused={false}
				isHovered={false}
				isActive={false}
				isPlain={false}
				isDisabled={false}
				isPrimary={false}
				iconComponent={() => <p>ReactElementType</p>}
				splitButtonItems={[]}
				splitButtonVariant={'checkbox'}
				aria-label={"string"}
				ariaHasPopup={true}
				type={'button'}
				onEnter={(event?: React.MouseEvent<HTMLButtonElement>) => undefined as void}
      />);
    expect(view).toMatchSnapshot();
  });
});
