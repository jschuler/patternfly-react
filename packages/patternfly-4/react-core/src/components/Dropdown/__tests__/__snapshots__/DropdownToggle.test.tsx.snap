// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DropdownToggle test DropdownToggle should match snapshot 1`] = `
<div
  className="pf-c-dropdown__toggle pf-m-split-button"
>
  <ContextConsumer>
    <Component />
  </ContextConsumer>
</div>
`;
